code/modules/food/drinkingglass/metaglass.dm

```diff
@@ -670,4 +670,4 @@ Drinks Data
 	glass_center_of_mass = list("x"=16, "y"=8)
 
 /datum/reagent/ethanol/mojito
-	glass_icon_state = "mojito"
\ No newline at end of file
+	glass_icon_state = "mojito"
```

Comment:
```suggestion
	glass_icon_state = "mojito"```
Let's, uhhh, see if this comment works for the newline issue...?

Reply:
I'd leave it as-is. Don't mess with it further, tbh, unless you want to do a revert and dismiss changes, but that's a little complex for a newbie.